docs/content/contributing/bicep/_index.md

```diff
@@ -34,10 +34,24 @@ Before you start contributing to the AVM, it is **highly recommended** that you
 To contribute to this project the following tooling is required:
 
 - [Git](https://git-scm.com/downloads)
+    
+  If just installed, don't forget to set both your git username & password
+    
+    ```PowerShell
+    git config --global user.name "John Doe"
+    git config --global user.email "johndoe@example.com"
+    ```
+
 - [Bicep](https://learn.microsoft.com/azure/azure-resource-manager/bicep/install#install-manually)
+
+  {{< hint type=note >}}
+
+  Must be manually kept up-to-date.
+
+  {{< /hint >}}
+
 - [Visual Studio Code](https://code.visualstudio.com/download)
   - [Bicep extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep)
-  - [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig)
```

Comment:
@eriqua We need this one. As it's used for linting and enforcing spaces, tabs and white space etc in the repo

Reply:
Removed as duplicated below in the _recommended_ tools. If you vouch for that to be required I'll remove it from the next list instead

Reply:
I would like it recommended as folks Will get linting errors otherwise more commonly.

Reply:
I agree the extension is useful. We can either
1) List that in the recommended tools and remove it from required tools (proposed change) -OR-
2) List that in the required tools and remove it from recommended tools 

I don't have a strong preference, but I'd avoid having the same extension listed twice as it is currently.

Reply:
Okay lets leave as recommended 👍